src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -1013,14 +1012,13 @@ void SolverInterfaceImpl::mapReadDataTo(
       mappingContext.mapping->computeMapping();
     }
     for (impl::DataContext &context : _accessor->readDataContexts()) {
-      if (context.mesh->getID() != toMeshID) {
+      if (context.getMeshID() != toMeshID) {
         continue;
       }
-      context.toData->values() = Eigen::VectorXd::Zero(context.toData->values().size());
-      PRECICE_DEBUG("Map data \"{}\" to mesh \"{}\"", context.fromData->getName(), context.mesh->getName());
-      PRECICE_ASSERT(mappingContext.mapping == context.mappingContext.mapping);
-      mappingContext.mapping->map(context.fromData->getID(), context.toData->getID());
-      PRECICE_DEBUG("Mapped values = {}", utils::previewRange(3, context.toData->values()));
```

Comment:
Looks like an accident. Will add it again.

Reply:
Added the debug message again in dfa4d356ae31ae0d83403bdcc7abe18bfda82863. I now remember why I removed it: Because it allowed me to not provide any access to `DataContext::_toData`. If we want this debug message, we have to provide access and this makes the interface more complicated. I added a `@todo` to point to this.

I would rather not touch this point here, since it requires to refactor the mapping class hierarchy or introduce lots of code duplication, if we want to move this debug message into the mapping (where accessing `_toData` should not be any problem).